packages/phpstan-rules/src/Rules/ForbiddenFuncCallRule.php

```diff
@@ -55,7 +55,7 @@ public function process(Node $node, Scope $scope): array
             return [];
         }
 
-        if (! $this->arrayStringAndFnMatcher->isMatch($funcName, $this->getForbiddenFunctionsList())) {
+        if (! $this->arrayStringAndFnMatcher->isMatch($funcName, array_keys($this->getForbiddenFunctionsWithMessages()))) {
```

Comment:
Now the argument looks a bit unreadable. At first sight its array of integers.
What about extracting method that would return desired values without using `array_keys`?